packages/driver/src/cypress.ts

```diff
@@ -14,7 +14,7 @@ import browserInfo from './cypress/browser'
 import $scriptUtils from './cypress/script_utils'
 
 import $Commands from './cypress/commands'
-import $Cy from './cypress/cy'
+import { $Cy } from './cypress/cy'
```

Comment:
```suggestion
import { Cy } from './cypress/cy'
```

Reply:
The name of the internal class is `$Cy`, not `Cy`.